README.md

```diff
@@ -26,6 +26,100 @@
+
+### Tasks and components
+
+This is an overview of the tasks supported by the AllenNLP Models library along with the corresponding components provided, organized by category. For a more comprehensive overview, see the [AllenNLP Models documentation](https://docs.allennlp.org/models/main/) or the [Paperswithcode page](https://paperswithcode.com/lib/allennlp).
+
+- [**Classification**](https://github.com/allenai/allennlp-models/tree/main/allennlp_models/classification)
+  
+  Classification is a broad category that contains many more specific tasks such as Sentiment Analysis and Binary Question Answering.
+
+  🛠 **Components provided:** Dataset readers for various datasets, including [BoolQ](https://docs.allennlp.org/models/main/models/classification/dataset_readers/boolq/) and [SST](https://docs.allennlp.org/models/main/models/classification/dataset_readers/stanford_sentiment_tree_bank/), as well as a [Biattentive Classification Network](https://docs.allennlp.org/models/main/models/classification/models/biattentive_classification_network/) model.
+
+- [**Coreference Resolution**](https://github.com/allenai/allennlp-models/tree/main/allennlp_models/coref)
+
+  Coreference resolution is the task of finding all expressions that refer to the same entity in a text. It is an important step for many higher level NLP tasks that involve natural language understanding such as document summarization, question answering, and information extraction.
+
+  🛠 **Components provided:** A general [Coref](https://docs.allennlp.org/models/main/models/coref/models/coref/) model and several dataset readers.
+
+- [**Generation**](https://github.com/allenai/allennlp-models/tree/main/allennlp_models/generation)
+
+  This is a category for tasks such as Summarization that involve generating unstructered and often variable-length text.
+
+  🛠 **Components provided:** Several Seq2Seq models such a [Bart](https://docs.allennlp.org/models/main/models/generation/models/bart/), [CopyNet](https://docs.allennlp.org/models/main/models/generation/models/copynet_seq2seq/), and a general [Composed Seq2Seq](https://docs.allennlp.org/models/main/models/generation/models/copynet_seq2seq/), along with corresponding dataset readers.
+
+- [**Language Modeling**](https://github.com/allenai/allennlp-models/tree/main/allennlp_models/lm)
+
+  Language Modeling tasks involve learning a probability distribution over sequences of tokens.
+
+  🛠 **Components provided:** Several language model implementations, such as a [Masked LM](https://docs.allennlp.org/models/main/models/lm/models/masked_language_model/) and a [Next Token LM](https://docs.allennlp.org/models/main/models/lm/models/next_token_lm/).
+
+- [**Multiple Choice**](https://github.com/allenai/allennlp-models/tree/main/allennlp_models/mc)
+
+  Multiple Choice tasks are tasks that involve selecting a correct answer out of a set of possible answers, given some input.
```

Comment:
Personally confused on the distinction between `multiple choice` and `classification`

Reply:
Multiple choice is choosing between alternatives. Classification is evaluating each alternative on its own. In practice we can get human-level performance by classifying each option on its own, but I suspect that's a problem with the datasets. I wonder if a model that can actually compare possible answers against each other could do even better than what we do now.

Reply:
Just updated. Let me know if that makes sense.